benchmark/misc/string-creation.js

```diff
@@ -1,17 +0,0 @@
-'use strict';
-
-var common = require('../common.js');
-var bench = common.createBenchmark(main, {
-  millions: [100]
-});
-
-function main(conf) {
-  var n = +conf.millions * 1e6;
-  bench.start();
-  var s;
-  for (var i = 0; i < n; i++) {
-    s = '01234567890';
-    s[1] = 'a';
```

Comment:
Perhaps this line was added to prevent v8 from optimizing the for-loop away or something (since `s` wouldn't have been referenced)?